server/golobe/golobe/settings.py

```diff
@@ -31,6 +31,8 @@
 # Application definition
 
 INSTALLED_APPS = [
```

Comment:
This is really nice thank you tobi....Probably after we have the meeting tomorrow. You'll get a chance to speak with the other backend engineers